apps/tangle-dapp/src/data/liquidStaking/useAssetAccounts.ts

```diff
@@ -9,7 +9,7 @@ import { map } from 'rxjs';
 import useNetworkFeatures from '../../hooks/useNetworkFeatures';
 import { NetworkFeature } from '../../types';
 
-const useLsPoolMembers = ():
```

Comment:
For the _// TODO: Manually casting the type here, since the type is being inferred as `Codec` instead of `Option<PalletAssetsAssetAccount>`._ Sometimes it happens when you use `.entries()`, you can help TS augment the type by passing the result type to the generic `entries<ActualTypeHere>()`.